src/openforms/forms/models/form_step.py

```diff
@@ -54,6 +54,12 @@ class FormStep(OrderedModel):
     get_save_text = literal_getter("save_text", "form_step_save_text")
     get_next_text = literal_getter("next_text", "form_step_next_text")
 
+    literal_fields = (
```

Comment:
@sergei-maertens these are now obsolete due to your refactor for the literal_getters per language